xlators/cluster/dht/src/dht-rebalance.c

```diff
@@ -3439,7 +3439,7 @@ gf_defrag_process_dir(xlator_t *this, gf_defrag_info_t *defrag, loc_t *loc,
                                       migrate_data, dir_dfmeta, xattr_req,
                                       perrno);
 
-            if (defrag->defrag_status == GF_DEFRAG_STATUS_STOPPED) {
+            if (defrag->defrag_status != GF_DEFRAG_STATUS_STARTED) {
```

Comment:
Just curious about the change..what else can be caught here? GF_DEFRAG_STATUS_FAILED?

Reply:
Yes